src/lib.rs

```diff
@@ -49,6 +49,12 @@
 //! }
 //! ```
 //!
+//! Now, the above dispatcher is pretty easy to setup.
+//! There's also a more flexible and performant one;
+//! once you know about system data and parallelization,
```

Comment:
I think the ordering of sentences is bit confusing here, but I am not sure how to change it.
````rust
//! Once you are more familiar with how system data and parallelization works,
//! you can take look at more flexible and performant way to dispatch using `ParSeq`.
//! Using it is bit trickier, but it allows dispatching without any virtual function calls.
````
Maybe?